src/git/zcl_abapgit_git_porcelain.clas.abap

```diff
@@ -27,7 +27,6 @@ CLASS zcl_abapgit_git_porcelain DEFINITION
       IMPORTING
         !io_repo TYPE REF TO zcl_abapgit_repo_online
         !iv_name TYPE string
-        !iv_from TYPE zif_abapgit_definitions=>ty_sha1
```

Comment:
can we keep this parameter? Ideally, someday the git classes should be independent of ZCL_ABAPGIT_REPO_ONLINE. In the future, the user might be able to select what should be the base of the new branch

Reply:
Fixed.